DiceKeys/Views/Scanning/ScanDiceKey.swift

```diff
@@ -77,7 +77,11 @@ struct ScanDiceKey: View {
                         Spacer()
                             GeometryReader { reader in
                                 ZStack {
-                                    DiceKeysCameraView(selectedCamera: selectedOrNextBestCameraDisplayableCamera, onFrameProcessed: onFrameProcessed, size: reader.size)
+                                    #if os(iOS)
```

Comment:
I think this was fixed in #58 by adding selectedCamera to DiceKeysCameraView.init in iOS.

Reply:
Reverted the change